Common/UnitDefinitions/VolumeFlowPerArea.json

```diff
@@ -0,0 +1,46 @@
+{
+  "Name": "VolumeFlowPerArea",
+  "BaseUnit": "CubicMeterPerSecondPerSquareMeter",
+  "XmlDoc": "The volumetric flow rate per area is the volume of fluid which passes through a given unit surface area per unit time.",
+  "BaseDimensions": {
+    "L": 3,
+    "T": -1
+  },
+  "Units": [
+    {
+      "SingularName": "CubicMeterPerSecondPerSquareMeter",
+      "PluralName": "CubicMetersPerSecondPerSquareMeter",
+      "BaseUnits": {
+        "L": "Meter",
+        "T": "Second"
+
+      },
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "m³/s·m²" ]
```

Comment:
I am not sure which way is better to format this. What is used in your domain?
Not being familiar with the domain, I would suggest `m³/(s·m²)` as maybe more readable.

Reply:
It's mainly for building's infiltration and ventilation.
But `m³/(s·m²)` could also work.